### ISSUE TITLE
Updating from symfony 4 to 5 breaks with null handler

### ISSUE THREAD

**Issue by gmponos** [symfony/monolog-bundle#335]
I have upgraded my application from 4.4.1 to 5.0.1

inside my packages/test/monolog.yaml I have:
```
monolog:
    handlers:
        main:
            type: 'null'
            level: debug
```

When I run my tests I get `Error : Call to undefined method Monolog\Handler\NullHandler::pushProcessor()`

is this a bug or there is a workaround (besides setting test/debug handler)?

**Comment by acasademont** [comment]
I downgraded to monolog 1.X in the meantime, that should fix it.

By default, if you only have `monolog-bridge` in your composer.json, it will pick up the highest monolog version available (2.0.5). Explicitly requiring `monolog/monolog:^1.25` should do it

**Comment by tzunghaor** [comment]
Symfony tried to push psr formatter processor to my null handler, but explicitly disabling it solved my problem:

```yaml
monolog:
    handlers:
        main:
            type: 'null'
            process_psr_3_messages: false
```

**Comment by Seldaek** [comment]
The bundle should check whether ProcessableHandlerInterface is implemented (in Monolog 2) before calling pushProcessor.

**Comment by jderusse** [comment]
Fixed by #390
